addons/im_livechat/static/tests/embed/livechat_button_tests.js

```diff
@@ -22,10 +23,19 @@ test("open/close temporary channel", async () => {
 test("open/close persisted channel", async () => {
     await startServer();
     await loadDefaultConfig();
-    start();
+    await start({
+        mockRPC(route, args, originalRPC) {
+            const res = originalRPC(...arguments);
+            if (route === "/im_livechat/get_session" && args.persisted) {
+                step("persisted");
+            }
+            return res;
+        },
+    });
     await click(".o-livechat-LivechatButton");
     await insertText(".o-mail-Composer-input", "How can I help?");
-    triggerHotkey("Enter");
+    await triggerHotkey("Enter");
+    await assertSteps(["persisted"]);
```

Comment:
Wasn't posting the message supposed to guarantee the persistence?

The selector for message was taking account temporary message maybe?
Can't we ensure it is non-temporary/persisted in a functional way rather than technical?